openlibrary/macros/ReadButton.html

```diff
@@ -12,14 +12,14 @@
 $ stream_url = "/borrow/ia/%s?ref=ol" % ocaid
 
 <div class="cta-button-group">
-  <a href="$(stream_url)" title="$title" id="$(action)_ebook"
+  <a href="$(stream_url)" title="$title" class="$(action)_ebook cta-btn cta-btn--available"
```

Comment:
I'd recommend using the class name  `cta-btn--$(action)` here to keep with the BEM syntax we've been using.

Reply:
Updated, let me know if it looks fine :D